packages/ckeditor5-typing/src/utils/findattributerange.js

```diff
@@ -4,38 +4,44 @@
  */
 
 /**
- * @module link/findlinkrange
+ * @module typing/utils/findattributerange
  */
 
 /**
- * Returns a range containing the entire link in which the given `position` is placed.
+ * Returns a range containing the entire {@link module:engine/view/attributeelement~AttributeElement#childCount attribute element}
+ * in which the given `position` is placed.
```

Comment:
This statement is unfortunate. This will return `model` range and we do not have `AttributeElement` in model. Also, the link points to `#childCount`.